django-verdant/Dockerfile

```diff
@@ -38,10 +38,12 @@ EXPOSE 8000
 # Install operating system dependencies.
 RUN apt-get update -y && \
     apt-get install -y apt-transport-https rsync libldap2-dev libsasl2-dev && \
-    curl -sL https://deb.nodesource.com/setup_8.x | bash - &&\
-    apt-get install -y nodejs &&\
+    curl -sL https://deb.nodesource.com/setup_14.x | bash - && \
```

Comment:
**Not blocking:** There's a more modern way to do this now. We recently updated all our sites to match wagtail kit for the node installation.

Reply:
Cheers nick, I'll stick with this for now as we aren't going to be working on this or longer